tensorflow/python/eager/pywrap_tfe_test.py

```diff
@@ -142,7 +142,7 @@ def testFastpathExecute_TapeWrite_MatMul(self):
 
     dz_da = tape.gradient(z, [a_2_by_2])[0]
     self.assertAllEqual(dz_da.numpy(),
-                        constant_op.constant(2.0, shape=[2, 2]).numpy())
+                        constant_op.constant([[2., 2.], [2., 2.]]).numpy())
```

Comment:
I think you can replace constant_op.constant(foo).numpy() with just foo.

Reply:
I'll gladly change it, just want to double-check since all the other tests use the "constant_op.constant(foo).numpy()" syntax, so I was following them for consistency. Should I still change it?

Reply:
@alextp Can you please assist on above comments from @amturati. Thanks!

Reply:
This is just going through a bunch of extra trouble to convert a list into a tensor and then into a numpy array. I'm pretty sure it'll work just fine with the list as a constant.

Sometimes the existing code looks weird for no good reason (e.g. because it was incrementally refactored from before we had tensor.numpy, or before we had tests fully running in eager mode, etc) and we should strive to not replicate these obsolete patterns on new code whenever we can.